{{cookiecutter.project_slug}}/config/settings/local.py

```diff
@@ -48,6 +48,10 @@
 INSTALLED_APPS += ('debug_toolbar', )
 
 INTERNAL_IPS = ('127.0.0.1', '10.0.2.2',)
+# tricks to have debug toolbar when developing with docker
+import socket
+ip = socket.gethostbyname(socket.gethostname())
+INTERNAL_IPS += [ip, ip[:-1]+"1"]
```

Comment:
This looks good. I didn't get why are you using `ip[:-1]+"1"`. 

I think it would be nice, if you put the import statement on the top. My 2 cents!